demisto_sdk/commands/split/split_setup.py

```diff
@@ -12,9 +12,13 @@
 @logging_setup_decorator
 def split(
     ctx: typer.Context,
-    input: Path = typer.Option(..., help="The yml/json file to extract from"),
+    input: Path = typer.Option(
+        None, "-i", "--input", help="The yml/json file to extract from"
```

Comment:
```suggestion
        ..., "-i", "--input", help="The yml/json file to extract from"
```
Keep it as it was (it makes `input` mandatory)